src/Workload/maui.vsmanproj

```diff
@@ -0,0 +1,104 @@
+    <ManifestDropName Condition="'$(ManifestNameSuffix)' == ''">$(ManifestDropNameRoot)/$(ManifestBuildNumber)</ManifestDropName>
+    <ManifestDropName Condition="'$(ManifestNameSuffix)' != ''">$(ManifestDropNameRoot)/$(ManifestNameSuffix)/$(ManifestBuildNumber)</ManifestDropName>
+    <ManifestPublishUrl>https://vsdrop.corp.microsoft.com/file/v1/$(ManifestDropName);</ManifestPublishUrl>
+    <ManifestIntermediateOutputPath>$(OutputPath)\obj\$(MSBuildProject)</ManifestIntermediateOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <MergeManifest Include="$(ManifestOutputPath)\*.json">
+      <RelativeUrl>/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\CHS\*.json">
+      <RelativeUrl>/CHS/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\CHT\*.json">
+      <RelativeUrl>/CHT/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\CSY\*.json">
+      <RelativeUrl>/CSY/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\DEU\*.json">
+      <RelativeUrl>/DEU/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\ENU\*.json">
+      <RelativeUrl>/ENU/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\ESN\*.json">
+      <RelativeUrl>/ESN/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\FRA\*.json">
+      <RelativeUrl>/FRA/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\ITA\*.json">
+      <RelativeUrl>/ITA/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\JPN\*.json">
+      <RelativeUrl>/JPN/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\KOR\*.json">
+      <RelativeUrl>/KOR/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\PLK\*.json">
+      <RelativeUrl>/PLK/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\PTB\*.json">
+      <RelativeUrl>/PTB/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\RUS\*.json">
+      <RelativeUrl>/RUS/</RelativeUrl>
+    </MergeManifest>
+    <MergeManifest Include="$(ManifestOutputPath)\TRK\*.json">
+      <RelativeUrl>/TRK/</RelativeUrl>
+    </MergeManifest>
+  </ItemGroup>
+
+  <Import Project="$(SwixBuildTargets)"/>
+
+  <Target Name="SetXamarinDropVars" >
+    <Message Text="##vso[task.setvariable variable=XamarinDropName;isOutput=true]$(ManifestDropName)" Importance="high" />
+    <Message Text="##vso[task.setvariable variable=XamarinDropManifestName;isOutput=true]$(ManifestName).vsman" Importance="high" />
+    <Message Text="##vso[task.setvariable variable=XamarinDropManifestVersion;isOutput=true]$(ManifestBuildVersion)" Importance="high" />
+    <PropertyGroup>
+      <_DropVars>
+XamarinDropName=$(ManifestDropName)
```

Comment:
@pjcollins does this look good with Xamarin naming ? can we rename this?

Reply:
With the latest v4 of the MSI convert template this SetXamarinDropVars target is no longer used, so it should be safe to remove this entirely.